flow/include/flow/ThreadHelper.actor.h

```diff
@@ -743,7 +743,34 @@ Future<T> safeThreadFutureToFutureImpl(ThreadFuture<T> threadFuture) {
 	return threadFuture.get();
 }
 
-// The allow anonymous_future type is used to prevent misuse of ThreadFutures.
+// The removeArenaFromStandalone() actors simulate the behavior of DLApi. In this case,
+// the memory is not owned by the Standalone. If the `future` goes out of scope, subsequent
+// access to the memory via the returned standalone will be invalid.
+ACTOR template <typename T>
+Future<Standalone<T>> removeArenaFromStandalone(Future<Standalone<T>> future) {
+	try {
+		Standalone<T> _ = wait(future);
+		return Standalone<T>(future.get(), Arena());
+	} catch (Error& e) {
+		throw e;
```

Comment:
You don't need to catch and rethrow errors in an `ACTOR` unless you are somehow handling them. This is automatically taken care of by the `ACTOR` and the error will be put into the resulting future if needed.